### ISSUE TITLE
learn php . 

### ISSUE THREAD

**Issue by anonymous043044** [anonymous043044/startup-bvp#11]
try to explore as much as we  can . then we will stop exploring and then we will start our work . after that if we will need something to learn or need to know then we will learn that thing at that moment . deadline is tuesday 14 . ohk ..i working slow right now .. and may be till 16th march .....due to java .. but still trying .

**Comment by chellanivaibhav** [comment]
try your best , im also doing some work on my app.
Also please make comments from your bansaltushar account.
also when you make a comment do like this @vaibhavchellani  this will send me a notif and ill come to know that youve made an issue or comment

**Comment by bansaltush** [comment]
have u completed your php?

**Comment by chellanivaibhav** [comment]
yes i have almost done it , i have also done database part now i can fetch and store data from databse easily.
@bansaltush

**Comment by bansaltush** [comment]
i have  just completed half php .. i will just complete full then i  will move on to front end .. i will cover mysql later .. what u say ..

**Comment by chellanivaibhav** [comment]
I would like if you would start working on front end side by side , so that I could apply backend functions to it . This way we will be able to see some work.
Proceed if it's fine with you  .

**Comment by bansaltush** [comment]
ohk i agree .. with this . if u allow me . can i work on front end tomorrow . otherwise i will begin it right now

**Comment by chellanivaibhav** [comment]
Yes ofcourse , you don't need my permission , we are partners in this . If we fail , we fail together , if we succeed we succeed together .

**Comment by bansaltush** [comment]
ya ya sure .. bhai

**Comment by bansaltush** [comment]
i have made a commit .. check it

**Comment by chellanivaibhav** [comment]
#13 merged

**Comment by bansaltush** [comment]
it is still a issue ,, we will work over it
